include/circt/Dialect/FIRRTL/FIRRTLStructure.td

```diff
@@ -131,7 +132,8 @@ def FModuleOp : FIRRTLModuleLike<"module", [SingleBlock, NoTerminator]> {
   let arguments =
             (ins ConventionAttr:$convention,
                  ArrayRefAttr:$portLocations,
-                 DefaultValuedAttr<AnnotationArrayAttr, "{}">:$annotations);
+                 DefaultValuedAttr<AnnotationArrayAttr, "{}">:$annotations,
+                 OptionalAttr<SymbolRefArrayAttr>:$layers);
```

Comment:
Suggestion: Make it a required attribute, an empty list serves the same purpose as a missing attr.
```suggestion
                 DefaultValuedAttr <SymbolRefArrayAttr, "{}">:$layers);
```

Reply:
I switched over to use a variant of this suggestion in the rebase. Thanks!